_posts/2020-11-2-kubernetes-webhooks-made-easy-with-openfaas-part-2.md

```diff
@@ -0,0 +1,291 @@
+---
+title: "Kubernetes Webhooks made easy with OpenFaaS - Part 2"
+description: "In this post you'll learn how to write Kubernetes Admission webhooks using OpenFaaS functions - Part 2"
```

Comment:
Change to `Learn how to write your Kubernetes Admission Controllers with OpenFaaS functions`

Reply:
Done